en/scripting/modules/example-protobufjs.md

```diff
@@ -0,0 +1,120 @@
+# Example: Use protobuf.js with npm
+
+This section illustrates how to use the npm package in a Cocos Creator project by installing and using [protobufjs](https://www.npmjs.com/package/protobufjs).
+
+## Install Node.js
+
+The npm package management tool `npm` is included in the Node.js distribution. You can use it after installing Node.js.
+
+Confirm that the installation of `npm` is successful with the following command:
+
+```bash
+> npm -v
+# Possible output:
+# 6.14.9
+```
+
+## Install the npm Package
+
+Executing the following command in the project root directory
+
+```bash
+> npm install --save protobufjs
+```
+
+will install the npm package `protobufjs` into the `/node_modules` directory and write the dependencies for this package to the file `package.json`.
+
+The `package.json` file is an npm manifest file and needs to be included in version control.
+
+> Cocos Creator recommends that the automatically generated `package-lock.json` also be included in version control to ensure that the developers install the same version of the package.
+
+The `/node_modules` directory is generally not included in version control.
+
+Once `package.json` has documented the dependencies, the following commands can be executed directly to reinstall or install in other environments.
+
+```bash
+> npm install
+```
+
+### Expansion: Use npm Mirrors
+
+npm reads and downloads packages from the [official npmjs source](https://www.npmjs.com/) by default. Some countries or regions may have network issues that cause installation to fail or install too slowly, so it is recommended to fix this by switching mirrors.
+
+First, install the npm package [nrm](https://www.npmjs.com/package/nrm) globally.
+
+```bash
+> npm install -g nrm
+```
+
+> `-g` means global, which installs the npm package directly on the current computer, so you don't need to run it again as long as it is executed once.
+
+Check the following valid npm mirrors:
+
+```bash
+> npx nrm ls
+# Possible output:
+# * npm -------- https://registry.npmjs.org/
+# yarn ------- https://registry.yarnpkg.com/
+# cnpm ------- http://r.cnpmjs.org/
+# taobao ----- https://registry.npm.taobao.org/
+# nj --------- https://registry.nodejitsu.com/
+# npmMirror -- https://skimdb.npmjs.com/registry/
+# edunpm ----- http://registry.enpmjs.org/
+```
+
+You can choose the appropriate mirror according to the current region. The `taobao` mirror is a good choice for users in mainland China. Execute the following command to switch mirrors.
```

Comment:
You can choose
-->
Choose

Reply:
The `taobao` mirror is a good choice for users in mainland China. 

remove this line for the english docs....